thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/v2/AlertTaskRunnerV2.java

```diff
@@ -135,126 +172,144 @@ private void sendAnomalyReport() throws Exception {
         // fetch anomalies having id lesser than the watermark for the same function with notified = false & endTime > last one day
         // these anomalies are the ones that did not qualify filtration rule and got modified.
         // We should add these again so that these could be included in email if qualified through filtration rule
-        List<MergedAnomalyResultDTO> filteredAnomalies = anomalyMergedResultDAO
-            .findUnNotifiedByFunctionIdAndIdLesserThanAndEndTimeGreaterThanLastOneDay(functionId,
+        List<MergedAnomalyResultDTO> filteredAnomalies = anomalyMergedResultDAO.findUnNotifiedByFunctionIdAndIdLesserThanAndEndTimeGreaterThanLastOneDay(functionId,
                 lastNotifiedAnomaly, false);
         if (CollectionUtils.isNotEmpty(filteredAnomalies)) {
           mergedAnomaliesAllResults.addAll(filteredAnomalies);
         }
       }
+    }
+    // apply filtration rule
+    results = AlertFilterHelper.applyFiltrationRule(mergedAnomaliesAllResults, alertFilterFactory);
+
+    // only anomalies detected by default scheduler are sent
```

Comment:
Please add "; the anomalies detected during replay or given by users will not be sent" to make it clearer.